app/uk/gov/hmrc/gform/models/optics/FormModelVisibilityOptics.scala

```diff
@@ -54,10 +54,12 @@ case class FormModelVisibilityOptics[D <: DataOrigin](
     evalAndApplyTypeInfo(typeInfo)
   }
 
+  def booleanExprResolver = BooleanExprResolver(booleanExpr => evalIncludeIfExpr(IncludeIf(booleanExpr), None))
```

Comment:
Again, any reason for this to be a def? This should be private as well

Reply:
This can't be `private`, since it it accessed from outside. It can be `val` though.